litestar/_openapi/schema_generation/schema.py

```diff
@@ -603,6 +604,7 @@ def create_component_schema(
             openapi_type: The OpenAPI type, defaults to ``OpenAPIType.OBJECT``.
             title: The schema title, generated if not provided.
             examples: A mapping of example names to ``Example`` instances, not required.
+            extra: Extra kwargs to add/overwrite in the generated schema.
```

Comment:
Lets add a `.. versionadded::` directive here for the docs

Reply:
Added, but I'm not sure of the exact syntax.

Also, aren't these internal APIs (in `litestar._openapi` package) so is that really needed here?

Reply:
sorry you're right.. not sure what I was thinking